docs/fast_data/architecture.md

```diff
@@ -26,6 +26,15 @@ The Real-Time Updater is responsible for listening to the events concerning chan
 
 It is up to you to decide whether to make it execute any of these, based on the architecture you prefer to build. For example, if having your Projections up to date in a very short period of time is crucial for your business, you might want to skip the part where you check which Single Views should be updated, delegating this job to the Single View Trigger and decreasing the overhead of the RTU.
 
+### Projection Storer (PS)
+
+Projection Storer service is an alternative to the Real-Time Updater that can be employed when configuring Fast Data event-driven architectures. Its goal is to ingest, clean and filter change events, transforming them into projections records to be stored onto the storage system (e.g. database).
+It maintains the same interface of Real-Time Updater, that is it consumes [ingestion events](/fast_data/inputs_and_outputs.md#ingestion-message) and emits  [pr-update events](/fast_data/inputs_and_outputs.md#projection-update-message) once it has stored projection records on the storage system.
```

Comment:
I feel this may be misunderstood by having the same environment variables, config maps and functions.

It is better to precise we're talking about the I/O of the service with kafka